src/activities/CreateAreaMeasurement2D.ts

```diff
@@ -0,0 +1,86 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import type {
+    IActivityHandler,
+    IActivityContext,
+} from "@geocortex/workflow/runtime/IActivityHandler";
+import { MapProvider } from "@geocortex/workflow/runtime/activities/arcgis/MapProvider";
+import { activate } from "@geocortex/workflow/runtime/Hooks";
+import AreaMeasurement2D from "@arcgis/core/widgets/AreaMeasurement2D";
+import MapView from "@arcgis/core/views/MapView";
+
+export interface CreateAreaMeasurement2DInputs {
+    /**
+     * @description Unit system (imperial or metric) or specific unit used for area values.
+     */
+    areaUnit?: "metric" | "imperial" | "square-inches" | "square-feet" | "square-us-feet" | "square-yards" | "square-miles" | "square-meters" | "square-kilometers" | "acres" | "ares" | "hectares" | string;
+
+}
+
+export interface CreateAreaMeasurementOutputs {
+    measurement?: {
```

Comment:
OK, this is my final thought... I think we should not have the nested `measurement` object in the outputs. We should collapse it and just have its child properties as outputs.

That `measurement` object belongs to the view model and it isn't really ours to expose. And who knows what would happen if the workflow author modified it.

As a result this would be simpler for the author. They could just reference `$measure1.area`.

For assigning the result you should just be able to change line 81 to `...measure,`